app/main.py

```diff
@@ -1 +1,11 @@
 # write your code here
+def copy_file(cmd: str) -> None:
+    cmd_list = cmd.split()
+    if len(cmd_list) == 3 and cmd_list[0] == "cp":
+        _, existing_file, new_file = cmd_list
+    if existing_file != new_file:
+        with (
+            open(existing_file, "r") as file_in,
+            open(new_file, "w") as file_out
+        ):
+            file_out.write(file_in.read())
```

Comment:
Add blank lines between different logical operations to improve the code readability and delete unnecessary comments.